library/core/src/main/java/com/google/android/exoplayer2/C.java

```diff
@@ -976,6 +976,38 @@ private C() {}
    */
   public static final int NETWORK_TYPE_OTHER = 8;
 
+  /**
+   * Adaptation set's role and accessibility descriptor value.
```

Comment:
nit - Avoid DASH specific terms; there's no reason roles can't be used for other media formats. Would suggest documenting like:
```
Track role flags. Possible values are {@link XXX}, ...
```
See how `SelectionFlags` is documented for an example.